src/client/components/VoteSession.js

```diff
@@ -107,6 +111,19 @@ const VoteSession = ({
     });
   }, [winnerArtworkId, data]);
 
+  useEffect(() => {
+    function showPosition(position) {
+      setLatitude(position.coords.latitude);
+      setLongitude(position.coords.longitude);
+    }
+    function getLocation() {
+      if (navigator.geolocation && data && data.online) {
+        navigator.geolocation.getCurrentPosition(showPosition);
+      }
+    }
+    getLocation();
+  });
```

Comment:
I think it would be good to declare a dependency of `useEffect` on `data.online` to avoid unnecessary re-evaluation (maybe even re-render??).

```js
useEffect(() => {
  // ...
},[data?.online])
```

I'm using optional chaining here, which is supported since Chrome 80 and Firefox 76 but afaict is part of our Babel build configuration as well.